kuma/wiki/content.py

```diff
@@ -559,6 +565,27 @@ def add_to_attr(attr_name, add_list):
             yield token
 
 
+class PreNotranslateClassFilter(html5lib_Filter):
+    """
+    Filter which annotates <pre> tags with the 'notranslate' class.
+    """
+
+    def __iter__(self):
+        input = html5lib_Filter.__iter__(self)
+        for token in input:
+            if token["type"] == "StartTag" and token["name"] == "pre":
+                attrs = dict(token["data"]) or {(None, "class"): ""}
+                for (namespace, name), value in attrs.copy().items():
+                    if name == "class" and "notranslate" not in value:
```

Comment:
Nit: This would miss annotating `pre` tags with a custom class name that is a superset of `notranslate`, like `notranslate:yada`, so it would be better to be exact with:
```suggestion
                    if name == "class" and "notranslate" not in value.split():
```

Reply:
Fixed.